sale_last_price_info/tests/test_sale_last_price_info.py

```diff
@@ -0,0 +1,28 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import odoo.tests.common as common
+
+
+class TestSaleLastPriceInfo(common.TransactionCase):
+    def setUp(self):
+        super(TestSaleLastPriceInfo, self).setUp()
+        self.sale_order_model = self.env["sale.order"]
+        self.sale_line_model = self.env["sale.order.line"]
+        self.sale_order = self.env.ref("sale.sale_order_4")
+        self.sale_line = self.env.ref("sale.sale_order_line_9")
+        self.partner = self.env.ref("base.res_partner_3")
+        self.product = self.env.ref("product.product_delivery_02")
+        self.price_unit = 100.0
+
+    def test_sale_last_price_info_demo(self):
+        sale_line = self.sale_line_model.search(
```

Comment:
It's better to rely on new records created in test. Then, don't search but rely on the record values you've just created.